tests/test_psnr.py

```diff
@@ -0,0 +1,81 @@
+import torch
+import pytest
+from skimage.metrics import peak_signal_noise_ratio
+
+from photosynthesis_metrics import psnr
+
+
+@pytest.fixture(scope='module')
+def prediction() -> torch.Tensor:
+    return torch.rand(4, 3, 256, 256)
+
+
+@pytest.fixture(scope='module')
+def target() -> torch.Tensor:
+    return torch.rand(4, 3, 256, 256)
```

Comment:
Please check with at least several sized on tensors.
Also check edge cases to see what happens when `torch.zeros()`, `torch.ones()`, empty tensors and things like that are passed